### ISSUE TITLE
[Cache] : vendor\/ezyang\/htmlpurifier\/library\/HTMLPurifier\/DefinitionCache\/Serializer not writable

### ISSUE THREAD

**Issue by guelosuperstart** [stevebauman/hypertext#2]
Hello,

When i tried to use you tool, i have this error : 

php 8.2
Symfony 6.3

```php
use Stevebauman\Hypertext\Transformer;
$text = (new Transformer())->toText($body);
{"error":"User Warning: Directory \/vendor\/ezyang\/htmlpurifier\/library\/HTMLPurifier\/DefinitionCache\/Serializer not writable. "}
```

Thank you

**Comment by LAGGOUNE-Walid** [comment]
Same issue

**Comment by stevebauman** [comment]
Thanks for the report -- this is to do with the underling HTML Purifier package requiring write access to cache its definition file. I'll try to have a fix out tomorrow 🙏

**Comment by FredrikAnkarang** [comment]
Any updates on this issue?

**Comment by stevebauman** [comment]
Sorry for the lateness on this fix! Can you please update to [v1.0.1](https://github.com/stevebauman/hypertext/releases/tag/v1.0.1) and try again? This should now be resolved 🙏

**Comment by guelosuperstart** [comment]
Hello @stevebauman ,

i tried this morning and now it's work well 👌 

Thank you so much.
